### DISCUSSION TITLE
how to disable density fill

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD#3448] [Q&A] [answered]
I have `USE_FILL = NO` and I still see that the `density_fill.tcl` script is run and it result into the following error 
```
[ERROR STA-0511] cannot open './results/myplatform/mydesign/base/5_route.sdc'.
Error: density_fill.tcl, 2 STA-0511
Command exited with non-zero status 1
```
There is no other info in the log that could help me figure out what is going on.
Is there a way to temporarily disable this step?

**Comment by maliberty** [comment, marked as answer]
The check is
```
ifneq ($(USE_FILL),)
```
so set it to nothing rather than NO

**Reply by QuantamHD** [reply]
That's pretty confusing @maliberty.

**Reply by oharboe** [reply]
> That's pretty confusing @maliberty.

There are many things in ORFS that could do with streamlining.

I would like to have a single source of truth for all variables and their default value as well as possible values and an explanation. Currently default values can be in C++, Tcl or make .mk and Makefile... Also, it would be good with a standarized true/false scheme...

**Reply by maliberty** [reply]
I agree though USE_FILL is unset by default so you have to go out of your way to set it.  Once you take a value you'll have to deal with TRUE/1/ON/ENABLE/etc.  Set or unset seems as good as anything for a boolean.
